strawberryfields/gbs/plot.py

```diff
@@ -343,3 +343,51 @@ def points(
         f = go.Figure(data=[p], layout=layout)
 
     return f
+
+
+LIGHT_BLUE = "#94cecf"
+
+BAR_COLOUR = LIGHT_BLUE
+
+
+def spectrum(energies: list, bins: Optional[int] = 0):  # pragma: no cover
+    """Plots a spectrum based on input sampled energies.
+
+    Args:
+        energies (list[float]): a list of sampled energies
+        bins (int): maximum number of bins. If zero, the number of bins is decided automatically
+            by Plotly.
+
+    Returns:
+         Figure: spectrum in the form of a histogram of energies
+    """
+    try:
+        import plotly.graph_objects as go
+    except ImportError:
+        raise ImportError(plotly_error)
+
+    h = go.Histogram(x=energies, nbinsx=bins, marker={"color": BAR_COLOUR})
```

Comment:
I think it might be good to make the limits of the x optional with some default values, such this:
x_start = 0.0
x_end = 8000.0
xbins = dict(start=x_start, end=x_end, size=0.5)

Reply:
Nice idea, similar things are done for `np.histogram` or `matplotlib.pyplot.hist`:
https://docs.scipy.org/doc/numpy/reference/generated/numpy.histogram.html
https://matplotlib.org/3.1.1/api/_as_gen/matplotlib.pyplot.hist.html